site/source/tutorials/1030_routing/index.md

```diff
@@ -76,51 +78,62 @@ Next, we will create `outlets` to control when our widgets are displayed.
 
 {% aside 'Reminder' %}
 The path that is associated to an outlet name is defined by the routing configuration from the first section of this tutorial.
 {% endaside %}
 
-An `Outlet` is a higher order component that wraps a widget, and controls whether the widget is rendered based on the navigated route associated to the outlets name. To keep our application organized, outlets are usually stored in a separate directory named `outlets`.
+The `Outlet` widget accepts two properties `id` and `renderer`, the `id` is the `outlet` from the routing configuration and the `renderer` is a function that returns widgets and nodes to display using `v()` and `w()`.
 
-{% instruction 'Add the following code to `WorkerFormOutlet.ts`.' %}
+The `renderer` function receives a `MatchDetails` object that provides information about the route match.
 
-{% include_codefile 'demo/finished/biz-e-corp/src/outlets/WorkerFormOutlet.ts' %}
+ * router: The router instance, which can be used to generate links with, for example, `router.getLine('outlet-id')`.
+ * queryParams: An object that contains any query params for the route.
+ * params: An object that contains any path params for the route.
+ * type: The type of match:
+   * `index`: An exact match
+   * `partial`: The route is a match but not exact
+   * `error`: The route is an error match
+ * isError: Helper function that returns true when the match type is error
+ * isExact: Helper function that returns true when the match type is exact
 
-The first argument for the `Outlet` function determines the widget to display when the configured route is selected. Consider an `outlet` configured for a `path` of `about`, its wrapped widget will render for a selected route `about` (described as an `index` match). The widget will also display for any route that the outlet's `path` partially matches, for example, `about/company` or `about/company/team`.
+Consider an `outlet` configured for a `path` of `about`, the widget that it returns from the `renderer` will render for a selected route `about` (described as an `index` match). The widget will also display for any route that the outlet's `path` partially matches, for example, `about/company` or `about/company/team`.
 
-Normally using a component as the first argument of `Outlet` is perfectly acceptable, however there are scenarios where it is necessary to explicitly define a widget for an `index` match. To support this more advanced configuration, the first argument also accepts an object that can be used to specify these components explicitly.
+Simply returning the widget or nodes that need to be displayed when an outlet has matched is usually all that is required, however there are scenarios where it is necessary to explicitly define a widget for an `index` or `error` match. This is where the `matchDetails` comes is, using the information from here we can write some simple logic to determine which widget to render for each scenario.
```

Comment:
This is where the `matchDetails` comes is, using the information from here we can write some simple logic to determine which widget to render for each scenario. -> This is where `matchDetails` is beneficial. By using the information from `matchDetails`, we can create simple logic to determine which widget to render for each scenario.